### ISSUE TITLE
Fix odom code?

### ISSUE THREAD

**Issue by AritroSaha10** [AritroSaha10/bootstrapped-vex-v5#10]
After trying to run a tank drive in the simulator made by Rayhaan, it seems that the current PID code would probably not work for a tank drive. The functions that would likely not work include:
- `move` (Not completely sure about this one, would need to check IRL)
- `moveToPoint` (As consequence to `move` not working)

[Move to Point](https://github.com/AritroSaha10/bootstrapped-vex-v5/blob/master/src/driveSystems/drivetrainPID.cpp#L49) would specifically not work because the move function is meant to allow for strafing, something a tank drive / nonholonomic drive cannot do. 

A possible fix would likely be turning on the spot (something that is already done), calculating the delta from the current point and target, and applying the velocity to the motors directly, instead of using the move function.

**Comment by AritroSaha10** [comment]
Of course, I don't actually know if it does (or doesn't) work until I run the code on a real robot.

**Comment by AritroSaha10** [comment]
Fixed, only after almost half a year
